java/core/src/java/org/apache/orc/impl/filter/FilterFactory.java

```diff
@@ -71,9 +71,11 @@ public static BatchFilter createBatchFilter(Reader.Options opts,
     // 2. Process PluginFilter
     if (opts.allowPluginFilters()) {
       List<BatchFilter> pluginFilters = findPluginFilters(filePath, conf);
-      if (!pluginFilters.isEmpty()) {
-        LOG.debug("Added plugin filters {} to the read", pluginFilters);
-        filters.addAll(pluginFilters);
+      List<BatchFilter> allowFilters =
```

Comment:
Maybe we can move this logic into `findPluginFilters`?

Reply:
Please rename this to `pluginFilters` to minize the diff.